.devcontainer/configure.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Configure script to setup VSCode DevContainer Environment."""
+
+from importlib.machinery import SourceFileLoader
+import inspect
+import json
+import os
+import shlex
+import subprocess
+import sys
+
+# pylint: disable=g-import-not-at-top
+try:
+  from shutil import which
+except ImportError:
+  from distutils.spawn import find_executable as which
+# pylint: enable=g-import-not-at-top
+
+CURRENT_DIR = os.path.dirname(
+    os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
Is there a particular reason for using `inspect` here rather than the builtin `__file__`? Also it's not necessary for this PR, but please generally prefer using `pathlib` over `os.path` (demoed here)

```suggestion
CURRENT_DIR = pathlib.Path(__file__).resolve().parent
```

Reply:
Done